test/integration/test.ts

```diff
@@ -72,4 +72,26 @@ suite("Launch and onboarding", function() {
     const rows = await app.client.elements(".QueryResultTable-table tbody tr");
     assert.strictEqual(rows.value.length, 1);
   });
+
+  test("Switch between queries", async () => {
+    await app.client.click(".QueryList-new i");
+    setValueToEditor("select 1;");
+    await app.client.click("ul.QueryList-list li:last-child");
+    const firstQuery = (
```

Comment:
should I wait for a while until dom changes?

Reply:
https://github.com/bdash-app/bdash/pull/108/commits/cb011413baf00f52f32d2ffb4d6d106c769cb2ab